store.html

```diff
@@ -0,0 +1,3160 @@
+                <option value="Wool">Wool</option>
+                <option value="Woolavington">Woolavington</option>
+                <option value="Wooler">Wooler</option>
+                <option value="Woolley">Woolley</option>
+                <option value="Woolpit">Woolpit</option>
+                <option value="Wootton">Wootton</option>
+                <option value="Worcester">Worcester</option>
+                <option value="Worcester Park">Worcester Park</option>
+                <option value="Worcestershire">Worcestershire</option>
+                <option value="Workington">Workington</option>
+                <option value="Worksop">Worksop</option>
+                <option value="Worlaby">Worlaby</option>
+                <option value="Wormley">Wormley</option>
+                <option value="Worthing">Worthing</option>
+                <option value="Worton">Worton</option>
+                <option value="Wotton-under-Edge">Wotton-under-Edge</option>
+                <option value="Wouldham">Wouldham</option>
+                <option value="Wragby">Wragby</option>
+                <option value="Wrawby">Wrawby</option>
+                <option value="Wrea Green">Wrea Green</option>
+                <option value="Wrington">Wrington</option>
+                <option value="Writtle">Writtle</option>
+                <option value="Wrotham">Wrotham</option>
+                <option value="Wroughton">Wroughton</option>
+                <option value="Wroxall">Wroxall</option>
+                <option value="Wychbold">Wychbold</option>
+                <option value="Wye">Wye</option>
+                <option value="Wylam">Wylam</option>
+                <option value="Wymeswold">Wymeswold</option>
+                <option value="Yalding">Yalding</option>
+                <option value="Yapton">Yapton</option>
+                <option value="Yarm">Yarm</option>
+                <option value="Yarmouth">Yarmouth</option>
+                <option value="Yarnton">Yarnton</option>
+                <option value="Yate">Yate</option>
+                <option value="Yateley">Yateley</option>
+                <option value="Yatton">Yatton</option>
+                <option value="Yaxley">Yaxley</option>
+                <option value="Yeadon">Yeadon</option>
+                <option value="Yealmpton">Yealmpton</option>
+                <option value="Yelverton">Yelverton</option>
+                <option value="Yeovil">Yeovil</option>
+                <option value="Yetminster">Yetminster</option>
+                <option value="York">York</option>
+                <option value="Youlgreave">Youlgreave</option>
+                <option value="Yoxall">Yoxall</option>
+              </select>
+            </div>
+            <div class="field-4">
+              <label for="postcode">Postcode</label>
+              <input required type="text" name="postcode" />
+            </div>
+          </div>
+          <h2>select a colour</h2>
```

Comment:
There should be a `fieldset` wrapping this set of radio buttons. This `h2` should be a `legend` tag instead.